x-pack/plugins/synthetics/public/legacy_uptime/components/settings/add_connector_flyout.tsx

```diff
@@ -26,18 +22,6 @@ interface KibanaDeps {
   triggersActionsUi: TriggersAndActionsUIPublicPluginStart;
 }
 
-export const ALLOWED_ACTION_TYPES: ActionTypeId[] = [
-  '.slack',
-  '.pagerduty',
-  '.server-log',
-  '.index',
-  '.teams',
-  '.servicenow',
-  '.jira',
-  '.webhook',
-  '.email',
-];
```

Comment:
we need this filtering like discussed in slack because we auto generate params for supported connectors for one click alert.

Reply:
Hi @shahzad31! This allowlist has been replaced by the `supportedFeatureIds` list, which is added to each connector type. You will see in each built in action type listed above, that the `UptimeFeatureId` is included. This means that the list of connectors and connector types shown in the Uptime settings page will still only be filtered to the connectors that are in this list.

LMK if that makes sense. Happy to discuss further!